requirements/base.txt

```diff
@@ -2,11 +2,11 @@ chevron~=0.12
 click~=7.1
 Flask~=1.1.2
 #Need to add Schemas latest SDK.
-boto3~=1.14.0, >=1.14.23
+boto3~=1.15.0
```

Comment:
Since we found why there was conflict before, we can revert this one

Reply:
or bump it to 1.16

Reply:
What happened here? Can it be bumped to 1.16 now?

Reply:
it had conflict with samtranslator's boto3 requirements. It was fixed.


> Can it be bumped to 1.16 now?

yes we will. Is there any reason you want us to bump this version?

Reply:
I understand. The reason I am asking is that by the time this change was made, boto3 was at 1.16.x. This in turn puts us in a position where we need to either use an older version of `aws-sam-cli` (which is not ideal) or an older version of `boto3` (also not ideal).

Reply:
Oh I see. Are you using pip to install sam-cli? We recommend to use homebrew (macOS, linux) or installer (windows) to install it. In that case, you would not have this dependencies issue.

Reply:
https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html

Reply:
Besides, I created a PR to loose the boto3's version https://github.com/aws/aws-sam-cli/pull/2493